### ISSUE TITLE
10.6.0 active devices show my others containers

### ISSUE THREAD

**Issue by lelamamalgache** [jellyfin/jellyfin-web#1436]
![image](https://user-images.githubusercontent.com/58141187/85056752-109f9900-b1a0-11ea-96b0-a2d94b13ed01.png)

**Expected behavior**
hide others containers in the active devices

**System (please complete the following information):**
 - OS: docker
 - Browser: Chrome
 - Jellyfin Version: 10.6.0

**Comment by Artiume** [comment]
Is it because you have an API connected? Kinda weird to see sonarr there

**Comment by lelamamalgache** [comment]
yes i see all containers which use the api of jellyfin, it's weird, sometimes i see them, sometimes not, but 95% i can't see them in "active devices"
when i check my devices (dashboard - devices), i can't find them (containers)

that not block me, i just want to share this weird thing with you

**Comment by MrTimscampi** [comment]
This would be a server-side change.

@jellyfin/core Can you move this to jellyfin/jellyfin, please?
